packages/perf/lib/modular/index.js

```diff
@@ -0,0 +1,70 @@
+import { isBoolean } from '@react-native-firebase/app/lib/common';
+import { firebase } from '..';
+
+/**
+ * Returns a Performance instance for the given app.
+ * @param app - FirebaseApp. Optional.
+ * @returns {Performance}
+ */
+export function getPerformance(app) {
+  if (app) {
+    return firebase.app(app.name).perf();
+  }
+
+  return firebase.app().perf();
+}
+
+/**
+ * Returns a Performance instance for the given app.
+ * @param app - FirebaseApp. Required.
+ * @param settings - PerformanceSettings. Set "dataCollectionEnabled" which will enable/disable Performance collection.
+ * @returns {Performance}
+ */
+export async function initializePerformance(app, settings) {
+  const perf = firebase.app(app.name).perf();
+
+  if (settings && isBoolean(settings.dataCollectionEnabled)) {
+    await perf.setPerformanceCollectionEnabled(settings.dataCollectionEnabled);
+  }
+
+  return perf;
+}
+
+/**
+ * Determines whether performance monitoring is enabled or disabled.
+ * @param perf - Performance instance
+ * @returns {Boolean}
+ */
+export function isPerformanceCollectionEnabled(perf) {
+  return perf.isPerformanceCollectionEnabled;
+}
+
+/**
+ * Returns nothing.
+ * @param perf - Performance instance
+ * @param enabled - A Boolean for setting Performance collection
+ * @returns {void}
+ */
+export function setPerformanceCollectionEnabled(perf, enabled) {
```

Comment:
and this

Reply:
good point - the JS web API for modular exposes the javascript properties and the initialize method but not this specific getter or a setter - should remove, I think you are right